lib/mix/tasks/phx.gen.release.ex

```diff
@@ -153,6 +160,27 @@ defmodule Mix.Tasks.Phx.Gen.Release do
     |> Map.new()
   end
 
+  defp auto_migrate?(app, test_opts) do
+    repos = Application.get_env(app, :ecto_repos) |> List.wrap()
+    Enum.any?(repos, fn repo ->
```

Comment:
Perhaps this should be `Enum.all?` so if multiple adapters are used, we don't automigrate?